EMML/6001-7000/EMML6770.xml

```diff
@@ -133,23 +91,28 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                 <keywords>
                     <term key="ChristianLiterature"/>
                     <term key="Hagiography"/>
-                    <term key="HistoryAndHistoriography"/>
-                    <term key="Liturgy"/>
                     <term key="Miracle"/>
-                    <term key="MonasticLiterature"/>
                 </keywords>
             </textClass>
-            <langUsage><language ident="en">English</language><language ident="gez">Gǝʿǝz</language></langUsage>
+            <langUsage><language ident="en">English</language></langUsage>
         </profileDesc>
         <revisionDesc>
             <change who="NV" when="2017-05-18">Created entity</change>  
          <change when="2018-11-30" who="NV">added titles</change>
+            <change when="2025-03-14" who="NV">updated the entity</change>
+            <change when="2025-05-21" who="NV">Corrected the nesting based on Eugenia Sokolinski's review</change>
         </revisionDesc>
     </teiHeader>
     <text xml:base="https://betamasaheft.eu/">
```

Comment:
after teiHeader you need the facsimile with graphic url as it is for all EMML stubs